src/data_info/clients/async_tasks.clj

```diff
@@ -30,3 +33,18 @@
 (defn get-by-filter
   [filters]
   (async-tasks-client/get-by-filter (config/async-tasks-client) filters))
+
+(defn paths-async-thread
```

Comment:
I moved this over here and named it this since it might be useful to be able to create these sorts of functions from several other namespaces, despite them basically being "path actions" in both places. We'd add other similar functions (or generalize this more) if we needed a different function signature for the update callback at some point.